web-app/js/smartR/heatmapService.js

```diff
@@ -10,24 +10,39 @@ HeatmapService = (function(smartRHeatmap){
         statusInterval : 0
```

Comment:
Could you please add nono minified version of jsrender? Asset pipeline will minify it for you.